src/flow-manager.c

```diff
@@ -264,6 +264,15 @@ static int FlowManagerFlowTimedOut(Flow *f, struct timeval *ts)
     }
 
     int server = 0, client = 0;
+
+    int state = SC_ATOMIC_GET(f->flow_state);
+    if (state == FLOW_STATE_BYPASSED)  {
+        if (FlowForceReassemblyNeedReassembly(f, &server, &client) == 1) {
```

Comment:
I don't see why this commit is needed. The flow will be handled by the flow timeout mechanism rather soon after the bypass, as it's timeout is now really low.